stories/ag/Drawer/Drawer.js

```diff
@@ -0,0 +1,60 @@
+import React from 'react'
+import { Drawer } from 'react-ui-ag/src'
+
+const DefaultDrawer = (
+  <div
+    style={
+      {
+        width: '300px',
+        height: '500px',
+        border: 'solid 1px black',
+        display: 'flex',
+        alignItems: 'flex-end',
+      }
+    }
+  >
+    <Drawer
+      openButtonContents={(<svg
+        xmlns="http://www.w3.org/2000/svg" width="24" height="24"
```

Comment:
Putting the svg inline in here is a bit overkill. See the RadioGroupDropdown for an example of how to do up and down chevrons in react-ui. Basically a unicode character.

Reply:
Changed